### ISSUE TITLE
Spacebar keys not rotated correctly

### ISSUE THREAD

**Issue by Merlin04** [riskable/keycap_playground#1]
This is easily fixable in the slicer but spacebar keys are not rotated to put the side flat on the build surface, so if you try to print them without fixing their rotation the print will likely fail.

![image](https://user-images.githubusercontent.com/11800751/108639598-44774080-744a-11eb-842f-819d3780f1d7.png)

**Comment by riskable** [comment]
Yeah for spacebars I usually have to rotate them manually.  I don't think there's anything I can do about it though because rotation value has just *one* default I can set and it obviously won't work for everything.
